web/tests/narrow_unread.test.js

```diff
@@ -30,6 +30,7 @@ people.add_active_user(alice);
 
 function set_filter(terms) {
     const filter = new Filter(terms);
+    filter.try_adjusting_for_moved_with_target();
```

Comment:
Normally, this is called in `message_view.js` after creating the filter. So we need to call it manually to get the terms adjusted here.